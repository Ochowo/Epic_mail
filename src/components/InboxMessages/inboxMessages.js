import React from 'react';

// eslint-disable-next-line react/prefer-stateless-function
class InboxMsg extends React.Component {
  render() {
    return (<div>heyyy</div>);
  }
}
export { InboxMsg };
export default InboxMsg;
