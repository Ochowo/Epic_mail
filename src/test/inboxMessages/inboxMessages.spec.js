import React from 'react';
import { shallow } from 'enzyme';
import { InboxMsg } from '../../components/InboxMessages/inboxMessages';

const props = {
  isAuthenticated: true,
  isLoading: false,
  auth: { isAuthenticated: true },
  messages: {
    message: { inbox: [
      {
        createdon: '2019-10-25T17:12:28.523Z',
        email: 'haruna@gmail.com',
        message: 'mmmm',
        messageid: 99,
        parentmessageid: null,
        receiverid: 16,
        ruserlastname: 'haruna',
        rusername: 'ochowo',
        senderid: 16,
        sfirstname: 'ochowo',
        slastname: 'haruna',
        status: 'unread',
        subject: 'nnn',
      },
    ] },

  },
  errors: null,
  getAllInbox: () => {},
};

describe('InboxMsg message component', () => {
  it('should render without errors', () => {
    const component = shallow(<InboxMsg {...props} />);
    expect(component).toMatchSnapshot();
  });
});
