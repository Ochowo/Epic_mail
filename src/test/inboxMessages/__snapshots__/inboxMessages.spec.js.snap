// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InboxMsg message component should render without errors 1`] = `ShallowWrapper {}`;
